### PR TITLE
Introduce a serif font theme option

### PR DESCRIPTION
**Pull request by cooltey**
https://phabricator.wikimedia.org/T264445

### CODE REVIEW

File: app/src/main/java/org/wikipedia/WikipediaApp.java [hunk 1]
```diff
@@ -301,6 +301,13 @@ public boolean setFontSizeMultiplier(int multiplier) {
         return false;
     }
 
+    public void setFontFamily(@NonNull String fontFamily) {
+        if (!fontFamily.equals(Prefs.getFontFamily())) {
+            Prefs.setFontFamily(fontFamily);
+            bus.post(new ThemeChangeEvent());
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/WikipediaApp.java:307]: May be we should rename it to something more generic like ThemeFontChangeEvent() or AppearanceChangeEvent()?

**Review by sharvaniharan — CHANGES_REQUESTED** [review]
Looks good.. I just wonder if we should show this only when we access the theme sheet from PageActivity..and not show it from settings screen, since this doesn't change overall app font, just article. <-question for @dbrant maybe

File: app/src/main/java/org/wikipedia/WikipediaApp.java (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/WikipediaApp.java]: Sounds good. I've updated to `ThemeFontChangeEvent()`.

File: app/src/main/java/org/wikipedia/analytics/AppearanceChangeFunnel.java [hunk 2]
```diff
@@ -3,16 +3,19 @@
...
 
 import org.json.JSONObject;
+import org.wikipedia.Constants.InvokeSource;
 import org.wikipedia.WikipediaApp;
 import org.wikipedia.dataclient.WikiSite;
 import org.wikipedia.theme.Theme;
 
 public class AppearanceChangeFunnel extends Funnel {
     private static final String SCHEMA_NAME = "MobileWikiAppAppearanceSettings";
     private static final int REV_ID = 18113727;
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/analytics/AppearanceChangeFunnel.java:13]: This needs to be a new revision ID, when Shay makes the schema changes.

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/analytics/AppearanceChangeFunnel.java (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/analytics/AppearanceChangeFunnel.java]: Done.

**Comment by cooltey** [comment]
Updated to the latest revision number
https://meta.wikimedia.org/wiki/Schema:MobileWikiAppAppearanceSettings

**Comment by sharvaniharan** [comment]
Lgtm... still feel. the 'Dark' theme could be lighter, but if Robin has okayed, that is good enough. @dbrant will let you merge. Also, any thoughts on my previous comment? 

>  I just wonder if we should show this only when we access the theme sheet from PageActivity..and not show it from settings screen, since this doesn't change overall app font, just article"

**Pull request merged**